### ISSUE TITLE
AC Transit re-import fails

### ISSUE THREAD

**Issue by drewda** [transitland/transitland-datastore#520]
https://dev.transit.land/dispatcher/feeds/f-9q9-actransit

feed version 0688399ae6b2a1458bf87f9af1e1c718d418e89b

exception log
```ruby
: {:exception_log=>"\n[428550:7 '(null)'] There are at least 8 columns in a row, which is beyond the number of provided column names (7).\n[\"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/rcsv-0.3.1/lib/rcsv.rb:137:in `raw_parse'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/rcsv-0.3.1/lib/rcsv.rb:137:in `parse'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/bundler/gems/gtfs-ec8d3cd10572/lib/gtfs/model.rb:83:in `block in each'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/bundler/gems/gtfs-ec8d3cd10572/lib/gtfs/model.rb:82:in `open'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/bundler/gems/gtfs-ec8d3cd10572/lib/gtfs/model.rb:82:in `each'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/bundler/gems/gtfs-ec8d3cd10572/lib/gtfs/source.rb:127:in `block (2 levels) in <class:Source>'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/bundler/gems/gtfs-ec8d3cd10572/lib/gtfs/source.rb:189:in `load_graph'\", \"/srv/www/transitland/releases/20160318164405/app/services/gtfs_graph.rb:28:in `create_change_osr'\", \"/srv/www/transitland/releases/20160318164405/app/workers/feed_eater_worker.rb:47:in `perform'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/processor.rb:151:in `execute_job'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/processor.rb:133:in `block (2 levels) in process'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/chain.rb:127:in `block in invoke'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-unique-jobs-4.0.17/lib/sidekiq_unique_jobs/lock/until_and_while_executing.rb:8:in `block in execute'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-unique-jobs-4.0.17/lib/sidekiq_unique_jobs/lock/while_executing.rb:19:in `synchronize'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-unique-jobs-4.0.17/lib/sidekiq_unique_jobs/lock/until_and_while_executing.rb:6:in `execute'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-unique-jobs-4.0.17/lib/sidekiq_unique_jobs/server/middleware.rb:16:in `call'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/chain.rb:129:in `block in invoke'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/server/active_record.rb:6:in `call'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/chain.rb:129:in `block in invoke'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/server/retry_jobs.rb:74:in `call'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/chain.rb:129:in `block in invoke'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/server/logging.rb:11:in `block in call'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/logging.rb:31:in `with_context'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/server/logging.rb:7:in `call'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/chain.rb:129:in `block in invoke'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/chain.rb:132:in `call'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/middleware/chain.rb:132:in `invoke'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/processor.rb:128:in `block in process'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/processor.rb:167:in `stats'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/processor.rb:127:in `process'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/processor.rb:79:in `process_one'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/processor.rb:67:in `run'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/util.rb:16:in `watchdog'\", \"/home/deploy/.bundler/transitland/ruby/2.2.0/gems/sidekiq-4.1.1/lib/sidekiq/util.rb:24:in `block in safe_thread'\"]\n"}
```

**Comment by irees** [comment]
Looks like there was a bad version of the feed temporarily posted to the website. The current file at the ACTransit Feed URL ( http://www.actransit.org/wp-content/uploads/gtfsDec202015B.zip ) has SHA1 e24abb4dfb496123ebc7345437c27415b0306296, the same as a Feed Version that has been previously successfully imported. The version at the next version of the URL ( http://www.actransit.org/wp-content/uploads/GTFSMar202016b.zip ) seems to import OK locally.

**Comment by irees** [comment]
Spoke to soon... checking.

**Comment by irees** [comment]
Fixed related bug in #523.

Closing this issue: error was a transient version of the file temporarily posted to agency url.
